recordset/recordset.js

```diff
@@ -112,8 +113,15 @@
                 $window.scrollTo(0, 0);
 
                 recordsetModel.page = page;
-                recordsetModel.rowValues = page.tuples.map(function (tuple, index, array) {
-                    return tuple.values;
+                recordsetModel.rowValues = page.tuples.map(function(tuple, index, array) {
+                    var row = [];
+                    tuple.values.forEach(function(value, index) {
+                        row.push({
+                            isHTML: tuple.isHTML[index],
+                            value: (tuple.isHTML[index]? $sce.trustAsHtml(value) : value)
+                        });
+                    });
+                    return row;
```

Comment:
I see this block of code duplicated 3 different times in the recordset controller. It might be cleaner to factor it out into a function of the controller.